.github/run_kuttl_tests.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -ex
+
+# Install kuttl
+if [[ ! -f /usr/local/bin/kubectl-kuttl ]]; then
+    wget -O /usr/local/bin/kubectl-kuttl https://github.com/kudobuilder/kuttl/releases/download/v0.9.0/kubectl-kuttl_0.9.0_linux_x86_64
+    chmod +x /usr/local/bin/kubectl-kuttl
+fi
+
+
+OPERATOR=$(echo ${1} | cut -d '-' -f1)
+export ${OPERATOR^^}_IMG=$2/$1-index:$3
+
+echo $4
+echo $5
+export ${OPERATOR^^}_REPO=https://github.com/$4.git
+echo ${OPERATOR^^}_REPO
+export ${OPERATOR^^}_BRANCH=$5
+echo ${OPERATOR^^}_BRANCH
+
+cd $HOME
+rm -rf install_yamls
+git clone https://github.com/openstack-k8s-operators/install_yamls.git
+cd $HOME/install_yamls
+make crc_storage
```

Comment:
Have you ever got an error on creating these storages? a failing pod? I added a loop on my patch[1], to avoid job to fail. needs to be investigated.

[1] https://github.com/openshift/release/blob/2fe7d9fccaaf60ed8cc5a3e6776f8332d5cfbdff/ci-operator/step-registry/openstack-k8s-operators/deploy/openstack-k8s-operators-deploy-commands.sh#L26-L38

Reply:
Yes, I have seen that kind of error in my local test but because I retry running the entire script, I haven't faced job failure due to those errors.

https://github.com/openstack-k8s-operators/openstack-k8s-operators-ci/pull/61/files#diff-f00fdc61987db9855cb64f3ae956149565247f8c8d48e0b2e31f4771ad7f1ca8R362-R365

~~~
        uses: nick-fields/retry@v2
        with:
          timeout_minutes: 10
          max_attempts: 5
~~~

I will add what you are doing.